src/app/components/BibPage/BibDetails.jsx

```diff
@@ -121,9 +122,9 @@ class BibDetails extends React.Component {
       return {
         term: fieldLabel,
         definition: (
-          <ul>
-            {bibValues.map((value, i) => <li key={i}>{value}</li>)}
-          </ul>
+          <span>
```

Comment:
So the structure will be <span> contains a series of <span> inside. Is this the expected structure?